rest/config.go

```diff
@@ -364,6 +364,8 @@ func (dbConfig *DbConfig) setDatabaseCredentials(credentials base.CredentialsCon
 // setup populates fields in the dbConfig
 func (dbConfig *DbConfig) setup(ctx context.Context, dbName string, bootstrapConfig BootstrapConfig, dbCredentials, bucketCredentials *base.CredentialsConfig, forcePerBucketAuth bool) error {
 	dbConfig.Name = dbName
+
+	// use db name as bucket if absent from config (handling for non-stamped configs)
```

Comment:
To clarify for future: Non stamped configs are configurations that are explicitly added with a REST api without `name` and also legacy configurations that don't have bucket in them?